docs/metrics/getting-started-prometheus-grafana/README.md

```diff
@@ -0,0 +1,196 @@
+see the metrics at the endpoint we've defined in `Program.cs`; in this case, the
+endpoint is: "http://localhost:9184/".
+
+Check the output metrics with your favorite browser:
+
+![MyFruitCounter output:](https://user-images.githubusercontent.com/16979322/150242010-8bde0002-44a5-4c84-94e6-3e0ee8a6ea4f.PNG)
+
+Now, we understand how we can configure Opentelemetry `PrometheusExporter` to
+export metrics the endpoint we sepcified. Next, we are going to learn about how
+to use Prometheus and Grafana to view/query our metrics with powerful
+visualization.
+
+## Download Prometheus
+
+Follow the [first steps]((https://prometheus.io/docs/introduction/first_steps/))
+to download the [latest release](https://prometheus.io/download/) of Prometheus.
+
+## Prometheus and Grafana
+
+### Configuration
+
+After finished downloading, extract it to a local location that's easy to
+access. We will find the default Prometheus configuration YAML file in the
+folder, named `prometheus.yml`.
+
+Let's create a new file in the same location as where `prometheus.yml` locates,
+and named the new file as `otel.yml` for this exercise. Then, copy and paste the
+entire content below into the otel.yml file we have created just now.
+
+```yaml
+global:
+  scrape_interval: 10s
+  scrape_timeout: 10s
+  evaluation_interval: 10s
+scrape_configs:
+- job_name: MyOpenTelemetryDemo
+  honor_timestamps: true
+  scrape_interval: 1s
+  scrape_timeout: 1s
+  metrics_path: /metrics
+  scheme: http
+  follow_redirects: true
+  static_configs:
+  - targets:
+  # set the target to the location where metrics will be exposed by
+  # the OpenTelemetry Prometheus Exporter
+    - localhost:9184
+```
+
+### Start Prometheus
+
+Follow the instructions from
+[starting-prometheus](https://prometheus.io/docs/introduction/first_steps/#starting-prometheus)
+to start the Prometheus server and verify it has been started successfully.
+
+Please note that we will need pass in otel.yml file as the argument:
+
+```console
+./prometheus --config.file=otel.yml
+```
+
+### View Results in Prometheus
+
+To use the graphical interface for viewing our metrics with Prometheus, navigate
+to "http://localhost:9090/graph", and type `MyFruitCounter` in the expression
+bar of the UI; finally, click the execute button.
+
+We should be able to see the following chart from the browser:
+
+![Prometheus Graph:](https://user-images.githubusercontent.com/16979322/150242083-65b84f25-c95f-4e9b-a64f-699ad8816602.PNG)
+
+From the legend, we can see that the `instance` name and the `job` name are the
+values we have set in `otel.yml`.
+
+Congratulations!
+
+Now we know how to configure Prometheus server and deploy OpenTelemetry
+`PrometheusExporter` to export our metrics. Next, we are going to explore a tool
+called Grafana, which has powerful visualizations for the metrics.
+
+### View/Query Results with Grafana
+
+First of all, please [Install Grafana](https://grafana.com/docs/grafana/latest/installation/).
```

Comment:
Do we want to add "First of all," here? 😂

Reply:
Not really :innocent:, removed.